install.ps1

```diff
@@ -1,84 +1,86 @@
 
 # Give time for user to cancel via CTRL+C
-Start-Sleep -s 3
+Start-Sleep -Seconds 3
 
-$checkSpice = Get-Command spicetify -ErrorAction Silent
-if ($null -eq $checkSpice) {
-  Write-Host -ForegroundColor Red "Spicetify not found. Installing that for you..."
-  Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" | Invoke-Expression
+if ( -not (Get-Command -Name spicetify -ErrorAction SilentlyContinue) ) {
+  Write-Host -Object "Spicetify not found. Installing it for you..." -ForegroundColor Red
+  Invoke-WebRequest -Uri "https://raw.githubusercontent.com/khanhas/spicetify-cli/master/install.ps1" -UseBasicParsing | Invoke-Expression
 }
 
 # Check if ~\.spicetify-cli\Themes\bloom directory exists
 $spicePath = spicetify -c | Split-Path
 $themePath = "$spicePath\Themes\bloom"
-if (-not (Test-Path $themePath)) {
-  Write-Host "Creating bloom theme folder..."
-  New-Item -Path $themePath -ItemType Directory | Out-Null
-} else {
+if (Test-Path -Path $themePath) {
   # Remove pre-existing files, only keep the newest files
-  Remove-Item "$themePath\*" -Recurse -Force
+  Remove-Item -Path "$themePath\*" -Recurse -Force
+}
+else {
+  Write-Host -Object "Creating bloom theme folder..."
+  New-Item -Path $themePath -ItemType Directory | Out-Null
 }
 
 # Download latest master
-$zipUri = "https://github.com/nimsandu/spicetify-bloom/archive/refs/heads/master.zip"
+$zipUri      = "https://github.com/nimsandu/spicetify-bloom/archive/refs/heads/master.zip"
 $zipSavePath = "$themePath\bloom-main.zip"
-Write-Host "Downloading spicetify-bloom latest master..."
+Write-Host -Object "Downloading spicetify-bloom latest master..."
 Invoke-WebRequest -Uri $zipUri -UseBasicParsing -OutFile $zipSavePath
 
 # Extract theme from .zip file
-Write-Host "Extracting..."
+Write-Host -Object "Extracting..."
 Expand-Archive -Path $zipSavePath -DestinationPath $themePath -Force
-Get-ChildItem "$themePath\spicetify-bloom-main\*" | ForEach-Object { Move-Item $_ $themePath }
-Remove-Item "$themePath\spicetify-bloom-main"
+Move-Item -Path "$themePath\spicetify-bloom-main\*" -Destination $themePath
+Remove-Item -Path "$themePath\spicetify-bloom-main"
 
 # Delete .zip file
-Write-Host "Deleting zip file..."
+Write-Host -Object "Deleting zip file..."
 Remove-Item -Path $zipSavePath
 
-# Change Directory to the Theme Folder
-Set-Location $themePath
-
 # Copy the bloom.js to the Extensions folder
-Copy-Item bloom.js ..\..\Extensions
-Write-Host "+ Installed bloom.js theme"
+Copy-Item -Path "$themePath\bloom.js" -Destination "$spicePath\Extensions"
+Write-Host -Object "Installed bloom.js theme"
 
 # Apply the theme with spicetify config calls
 spicetify config extensions bloom.js
 spicetify config current_theme bloom
 spicetify config color_scheme dark
 spicetify config inject_css 1 replace_colors 1 overwrite_assets 1
-Write-Host "+ Configured bloom theme"
+Write-Host -Object "Configured bloom theme"

```

Comment:
NO! Under any circumstances, DO NOT remove status prefixes! This will affect the UX and status prefixes are the easiest way to tell users if something went wrong!

Reply:
ok, I did this to match spicetify info messages style